resources/js/bots/transactor_personal/bot.js

```diff
@@ -321,18 +321,29 @@ function validateSend(params, context) {
 function handleSend(params, context) {
     var val = web3.toWei(params["amount"].replace(",", "."), "ether");
 
+    var gasPrice = calculateGasPrice(params["bot-db"]["sliderValue"]);
     var data = {
         from: context.from,
         to: context.to,
-        value: val,
-        gasPrice: calculateGasPrice(params["bot-db"]["sliderValue"])
+        value: val
     };
 
-    try {
-        return web3.eth.sendTransaction(data);
-    } catch (err) {
-        return {error: err.message};
+    if (gasPrice) {
+        data.gasPrice = gasPrice;
     }
+
+    web3.eth.sendTransaction(data, function(error, hash) {
+        if (error) {
+            // Do nothing, as error handling will be done as response to transaction.failed event from go 
```

Comment:
Can we print something? Standard out or debug stream

Reply:
I'm not sure how we can do that from js code executed in jail.
To be honest, whenever `web3.eth.sendTransaction` fails, the App receives signal `transaction.failed` from `status-go`, so I don't think it's necessary anyway.

Reply:
I think it's a dangerous assumption to assume that will always necessarily be the case, plus the specific error would be useful information when things go wrong, which they will.

That said, I looked at the other .js files and see no obvious way of printing to debug stream, since `log.debug` is commented out everywhere. There is some kind of debug mode but seems to be for console only, so maybe it's not trivial to do, so in that sense we can leave it alone.